Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/Amazon.Lambda.Annotations.SourceGenerator.csproj

```diff
@@ -3,6 +3,8 @@
   <PropertyGroup>
     <AssemblyVersion>1.4.0</AssemblyVersion>
     <TargetFramework>netstandard2.0</TargetFramework>
+    <AssemblyOriginatorKeyFile>..\..\..\buildtools\public.snk</AssemblyOriginatorKeyFile>
+    <SignAssembly>true</SignAssembly> 
```

Comment:
The `SQSEventAttribute` defined in `Amazon.Lambda.Annotations` contains some internal methods to indicate whether the properties of this attribute were set or not. The `Amazon.Lambda.Annotation.SourceGenerator` needs to access these internal methods. Both these assemblies need to be strongly signed for the `InternalsVisibleTo` to take effect.